rke/k8s_rke_system_images.go

```diff
@@ -3757,6 +3757,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.4"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled out of band post v2.4.11
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
+		"v1.18.13-rancher1-1": {
+			Etcd:                      m("rancher/coreos-etcd:v3.4.3-rancher1"),
+			Kubernetes:                m("rancher/hyperkube:v1.18.13-rancher1"),
+			Alpine:                    m("rancher/rke-tools:v0.1.66"),
```

Comment:
`1.18.13-rancher1-1` needs to be exactly the same in v2.4 and v2.5. This is so that clusters from v2.4 don't automatically get upgraded after rancher upgrade to v2.5. 

`rke-tools` in here is .66 vs in v2.5 is .67